src/similarity/similarity.py

```diff
@@ -0,0 +1,229 @@
+import librosa
+import fileinput
+from scipy.spatial import distance
+import numpy as np
+import os
+import falconn
+from multiprocessing import Pool, Process, Queue, cpu_count, Manager
+from database.song_segment import SongSegment
+
+MATCHES = 10
+BUCKET_SIZE = 60 * 100
```

Comment:
Could this be moved to the config file?

Reply:
Matches is probably a good idea, not sure about bucket size.